module/zfs/vdev_raidz.c

```diff
@@ -3484,6 +3790,10 @@ vdev_raidz_io_done(zio_t *zio)
 			for (int i = 0; i < rm->rm_nrows; i++) {
 				raidz_row_t *rr = rm->rm_row[i];
 				vdev_raidz_io_done_verified(zio, rr);
+
+				/* Periodically check for a read outlier */
+				if (zio->io_type == ZIO_TYPE_READ)
+					vdev_child_slow_outlier(zio);
```

Comment:
Why is this within the loop on `rm_nrows` if it does not take it as an argument?